logging/google/cloud/logging_v2/gapic/config_service_v2_client_config.py

```diff
@@ -3,7 +3,7 @@
         "google.logging.v2.ConfigServiceV2": {
             "retry_codes": {
                 "idempotent": ["DEADLINE_EXCEEDED", "INTERNAL", "UNAVAILABLE"],
-                "non_idempotent": [],
```

Comment:
This is Python, not JSON, dangit -- trailing commas in the last element of a list are a goodness!